plugins/lsp/language-servers/default.nix

```diff
@@ -591,7 +591,9 @@ let
           example = false;
         };
       };
-      extraConfig = cfg: { filetype.extension = mkIf cfg.autoSetFiletype { v = "vlang"; }; };
+      extraConfig = cfg: {
+        filetype.extension = mkIf (cfg.enable && cfg.autoSetFiletype) { v = "vlang"; };
```

Comment:
I wonder if the issue here is actually `_mk-lsp.nix` not having `mkIf cfg.enable` like the other mkPlugins do in our helper lib?